builtin/credential/cert/path_login.go

```diff
@@ -85,6 +78,86 @@ func (b *backend) pathLogin(
 	return resp, nil
 }
 
+func (b *backend) pathLoginRenew(
+	req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
+	config, err := b.Config(req.Storage)
+	if err != nil {
+		return nil, err
+	}
+
+	if config == nil || config.VerifyCert {
```

Comment:
Just a callback to the above comments about what comes out of b.Config and what's in a default config object...